compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/ScenarioServiceMain.scala

```diff
@@ -14,7 +14,8 @@ import scala.collection.JavaConverters._
 import scala.util.control.NonFatal
 
 object ScenarioServiceMain extends App {
-  val maxMessageSize = 128 * 1024 * 1024 // 128MB
+  // default to 128MB
+  val maxMessageSize = args.headOption.map(_.toInt).getOrElse(128 * 1024 * 1024)
```

Comment:
If we start piling up command line options here, we should a proper parser for it. Let's wait until then though.

Reply:
Yeah for now, I can’t be bothered to figure out CLI parsers in Scala :trollface: